util/options.py

```diff
@@ -262,10 +262,10 @@ def try_read_experiments_from_images(
             # Don't expand wildcards if URI-style filename
             if "*" in arg and not get_url_scheme(arg):
                 filenames = glob(arg)
-                if not filenames:
-                    logger.warning("No files found with pattern %s" % arg)
+                if filenames:
+                    args_new.extend(filenames)
                 else:
-                    args_new.extend(glob(arg))
+                    logger.warning("No files found with pattern %s" % arg)
```

Comment:
This is still the wrong way to use logging, as it should really either be:
`logger.warning("No files found with pattern %s", arg)` (which result in delayed formatting, which is only really needed if formatting is expensive)
or (my preference as f-String syntax is more modern/readable etc etc):
`logger.warning(f"No files found with pattern {arg}")`
Using `logger.warning("No files found with pattern %s" % arg)` gives the worst of both world as it uses the legacy `%` string formatting syntax, but without the potential advantages of delayed formatting within the logging library.

Reply:
Ah sorry I didn't realise there was an underlying difference between `%s", arg` and `%s" % arg`.